src/graphics/reproject-texture.js

```diff
@@ -40,7 +44,322 @@ function getProjectionName(projection) {
         case TEXTUREPROJECTION_EQUIRECT: return "Equirect";
         case TEXTUREPROJECTION_OCTAHEDRAL: return "Octahedral";
     }
+};
+
+// pack a 32bit floating point value into RGBA8
+const packFloat32ToRGBA8 = (value, array, offset) => {
```

Comment:
this should perhaps be moved to FloatPacking. And, please describe what the value range is: [0,1]

Reply:
Yes it should probably go there, but there are questions around naming and the fact it overlaps with `float2Bytes`. For this reason I'd prefer to move it in a different PR.